MoltenVK/MoltenVK/GPUObjects/MVKImage.mm

```diff
@@ -1489,6 +1489,19 @@
 	_componentSwizzle = pCreateInfo->components;
 	VkImageAspectFlags aspectMask = pCreateInfo->subresourceRange.aspectMask;
 
+	// Use swizzle adjustment to bridge some differences between Vulkan and Metal pixel formats.
+	// Do this ahead of other tests and adjustments so that swizzling will be enabled by tests below.
+	switch (pCreateInfo->format) {
+		case VK_FORMAT_BC1_RGB_UNORM_BLOCK:
+		case VK_FORMAT_BC1_RGB_SRGB_BLOCK:
+			// Metal doesn't support BC1_RGB, so force substituted BC1_RGBA alpha to 1.0.
+			_componentSwizzle.a = VK_COMPONENT_SWIZZLE_ONE;
```

Comment:
What happens if the swizzle's alpha component isn't `IDENTITY` or `A`? For that matter, what should happen if some other swizzle component is set to `A` here?

Reply:
Ah! Good catch. Thanks.

Updated to deal with both scenarios.